### ISSUE TITLE
CRAFT_NITRO/App::isNitro() doesn't work

### ISSUE THREAD

**Issue by timkelty** [craftcms/nitro#379]
Looks like it was added here, but then later removed:
https://github.com/craftcms/nitro/search?q=CRAFT_NITRO&type=commits

`App::isNitro()` will always return `false`, as `CRAFT_NITRO` isn't set (nginx or cli).

**Comment by jasonmccallister** [comment]
@timkelty I don't think we are going to add this feature back in and the upstream code is a little out-of-date.

**Comment by jasonmccallister** [comment]
Thoughts @brandonkelly?

**Comment by jasonmccallister** [comment]
@timkelty spoke with Brandon, going to transfer this from `nitro` to `cms` as we should remove this feature from core.
